aws/policy/application-services.yaml

```diff
@@ -134,6 +134,13 @@ Statement:
       - elasticfilesystem:DeleteMountTarget
       - kinesis:CreateStream
       - kinesis:DeleteStream
+      - glue:CreateJob
```

Comment:
This action incurs fees so should be under an appropriate Sid.

Reply:
Sorry about that, I didn't realize this! I was following the pricing description [here](https://aws.amazon.com/glue/pricing/), and took this statement as the `glue:CreateJob` action would not incur fees. 

> With AWS Glue, you only pay for the time your ETL job takes to run

I will move this to the correct Sid. Do you also want me to put these policy changes in `big-data.yml` instead of `application-services.yml`? I see there are another few `glue:*` action permissions in there.